### DISCUSSION TITLE
Upgrade jctools version?

### DISCUSSION THREAD

**Discussion by slandelle** [netty/netty#14095] [Ideas]
Netty currently uses a very old jctools version: 3.1.0, almost 4 years old.
Is there any reason for freezing the version?
AFAIK:
* it's shaded, hence doesn't conflict with the end-user's classpath
* jctools still emits Java 6 bytecode (there was a regression fixed in https://github.com/JCTools/JCTools/issues/392) so it's compatible with netty 4.1.

Thoughts?

**Comment by normanmaurer** [comment]
Sounds like a good idea to upgrade... PRs welcome

**Comment by slandelle** [comment]
on it

**Comment by slandelle** [comment]
See #14096
